pkg/controller/configmap/configmap_controller.go

```diff
@@ -139,8 +165,10 @@ func (r *ReconcileConfigMap) Reconcile(request reconcile.Request) (reconcile.Res
 			operatortypes.OperatorNamespace))
 		watchedNamespace = operatortypes.OperatorNamespace
 	}
-	log.Info(fmt.Sprintf("configmap controller watching events in namespace %s", watchedNamespace))
```

Comment:
It's better to move this line to place where request.Namespace == watchedNamespace